### PR TITLE
Implementing keybindings in the Menu.

### PR DESCRIPTION
**Pull request by BDisp**

### CODE REVIEW

File: Terminal.Gui/Core/Command.cs [hunk 1]
```diff
@@ -205,6 +205,41 @@ public enum Command {
 		/// <summary>
 		/// Moves the cursor to the bottom of page.
 		/// </summary>
-		EndOfPage
+		EndOfPage,
+
+		/// <summary>
+		/// Used to do some cleanup-
+		/// </summary>
+		CleanUp,
```
Comment (tznind) [inline Terminal.Gui/Core/Command.cs:213]: I don't think this makes sense as a Command.  Maybe put it back in explicitly as `ProcessKey` code or tie it to a `Leave` event or something?

Comment (BDisp) [reply Terminal.Gui/Core/Command.cs:213]: No needed to put it back. I'll test if putting in the `Leave` event works. Thanks.

Comment (BDisp) [reply Terminal.Gui/Core/Command.cs:213]: Doesn't work. How about changing it to `CloseItem`?

Comment (BDisp) [reply Terminal.Gui/Core/Command.cs]: I removed it because the `Toplevel` will process it. Thanks.

**Pull request merged**